charts/k8s-monitoring/charts/feature-integrations/integrations/grafana-values.yaml

```diff
@@ -0,0 +1,84 @@
+---
+# -- Name for this Loki instance.
+# @section -- General Settings
+name: ""
+
+# -- Discover Loki instances based on label selectors.
+# Will automatically set a matcher for `app.kubernetes.io/name: <name>` unless set here.
+# @section -- Discovery Settings
+labelSelectors: {}
+
+# -- The namespaces to look for Loki instances in.
+# Will automatically look for Loki instances in all namespaces unless specified here
+# @section -- Discovery Settings
+namespaces: []
+
+# -- Discover Loki instances based on field selectors.
+# @section -- Discovery Settings
+fieldSelectors: []
+
+# Settings for metrics collection
+metrics:
+  # -- Whether to enable metrics collection from Loki.
+  # @section -- Metrics Settings
+  enabled: true
+
+  # -- Name of the port to scrape metrics from.
+  # @section -- Discovery Settings
+  portName: grafana
+
+  # -- How frequently to scrape metrics from Loki.
+  # @default -- `60s`
+  # @section -- Scrape Settings
+  scrapeInterval:
+
+  # -- Sets the max_cache_size for prometheus.relabel component.
+  # This should be at least 2x-5x your largest scrape target or samples appended rate.
+  # ([docs](https://grafana.com/docs/alloy/latest/reference/components/prometheus.relabel/#arguments))
+  # Overrides global.maxCacheSize
+  # @default -- `100000`
+  # @section -- Metric Processing Settings
+  maxCacheSize:
+
+  # Adjustments to the scraped metrics to filter the amount of data sent to storage.
+  tuning:
+    # -- Filter the list of metrics from Grafana Loki to the minimal set required for the Grafana Loki integration.
+    # @section -- Metric Processing Settings
+    useDefaultAllowList: true
```

Comment:
Just like the empty file, if we don't have an allow list, then this should be removed.

Reply:
Removed